src/graph/GraphViewsManager.ts

```diff
@@ -92,7 +92,7 @@ class GraphViewDocumentContentProvider implements vscode.TextDocumentContentProv
     <!DOCTYPE html>
     <html>
       <body>
-        <iframe src="file://${clientHtmlPath}?port=${server.port}" width="100%" height="1000px" style="border:0"></iframe>
+        <iframe src="file://${clientHtmlPath}?port=${server.port}" style="width: calc(100% - 40px); height: 100%; position: absolute; padding: 0; margin: 0; border: none"></iframe>
```

Comment:
I'd rather not having sizing here (the 100%-40px) - that should be in the .css file.

Reply:
I tried to implement this in the .css, but I don't think the css properties were being applied to the class before the iframe would render.  It may have to do with the fact that the .css is loaded inthe graphClient.html, which is loaded within the iframe being created here.

Reply:
This is just for margins, right?  Let the iframe fill out the entire available space, and put the margins into the graphClient.html body or something, which can then be in the graphClient.css.